cli/tests/pcluster/config/test_param_to_cfn.py

```diff
@@ -30,6 +30,13 @@
         (SCALING, "scaledown_idletime", 10, "10"),
         (SCALING, "scaledown_idletime", 10, "10"),
         (SCALING, "scaledown_idletime", 3, "3"),
+        (
+            CLUSTER,
+            "extra_json",
+            {"cluster": {"cfn_scheduler_slots": "cores"}},
+            '{"cfncluster": {"cfn_scheduler_slots": "cores"}}',
+        ),
+        (CLUSTER, "extra_json", {"cluster": {"enable_ganglia": "true"}}, '{"cfncluster": {"enable_ganglia": "true"}}'),
```

Comment:
We could add another test by using "cfncluster" too as starting point.

Reply:
done